site/src/docs/components/header-universal-bar/code.mdx

```diff
@@ -0,0 +1,51 @@
+---
+slug: '/components/header-universal-bar/code'
+title: 'HeaderUniversalBar - Code'
+---
+
+import { Header, HeaderUniversalBar, NavigationLink } from 'hds-react';
+import ExternalLink from '../../../components/ExternalLink';
+import TabsLayout from './tabs.mdx';
+
+export default ({ children, pageContext }) => <TabsLayout pageContext={pageContext}>{children}</TabsLayout>;
+
+## Code
+
+### Code examples
+
+#### Default
+
+<InternalLink href="/components/navigation-link">Navigation links</InternalLink> can be put inside the header universal component indepedently as sibling elements or inside a container element.
+
+<Playground>
+
+```jsx
+<Header>
+  <HeaderUniversalBar primaryLinkHref="#" primaryLinkText="Helsingin Kaupunki">
```

Comment:
Should we always have the primaryLinkHref as [https://](https://www.hel.fi/fi) and primaryLinkText directing to Helsinki kaupunki?

Reply:
I was also thinking should we have the "https://hel.fi" and "Helsinki kaupunki" as default values in the component  ☝️

Reply:
I put these as default values. There is a problem with switching languages, as these should change as well, and this pattern leaves that problem for the user. If we would like to handle this problem ourselves so the user doesn't have to, we would need a (imo a dumb) prop `language` which would handle the link and text but that would mean that they are under our maintenance (right now hel.fi/en is a valid address but what if that changes and the language isn't carried in the route anymore?). What's your take on this?

Reply:
Actually this primary link should be removed. Source: https://helsinkicity.slack.com/archives/C0319GUE3RC/p1671190060005189